lambda/custom/APL/templates/titleMessageTemplate.js

```diff
@@ -0,0 +1,47 @@
+const layout = require('../layouts/titleMessageLayout.json');
+
+const template = ({ title, message }) => ({
+	type: 'Alexa.Presentation.APL.RenderDocument',
+	version: '1.0',
+	document: layout,
+	datasources: {
+
+		PostMessageData: {
+			type: 'object',
+			objectId: 'rcPostMessage',
+			backgroundImage: {
+				contentDescription: null,
+				smallSourceUrl: null,
+				largeSourceUrl: null,
+				sources: [
+					{
+						url: 'https://user-images.githubusercontent.com/41849970/60673516-82021100-9e95-11e9-8a9c-cc68cfe5acf1.png',
```

Comment:
Let's extract this and similar hardcoded URLs out in an APLConfig file and import the variable here and wherever it is used.
Also, text that will remain static through all the locales and screens can be put in the above config file.

And the dynamic text should come from (jargon+API) and passed to the template.

Please add your thoughts on this as well @PrajvalRaval.

Reply:
There's a translate function added to the handlerInput object which dynamically generates locale specific text and passes it to template